lib/calculator.rb

```diff
@@ -0,0 +1,25 @@
+require 'order'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.